### DISCUSSION TITLE
How should I provide attribution to this project when using it in my own mail client?

### DISCUSSION THREAD

**Discussion by jason-pezzimenti** [jstedfast/MailKit#1807] [Q&A]
^

**Comment by jstedfast** [comment]
The MIT license doesn't require you to provide any attribution in your app if you don't want to.

If you want to, though, you can. Typically, from what I've seen, apps that tend to provide attribution to the libraries that they use often seem to do it in their "About" dialog.

BTW, if you publish your mail client app, I would love to hear about it. It's always exciting to see stuff that developers are building with my libraries!

**Comment by jason-pezzimenti** [comment]
Thanks! Will let you know about it.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
